### PR TITLE
Create one test for each class

### PR DESCRIPTION
**Pull request by jcoyne**
Previously a bunch of job classes were all being tested in one spec file.
This removes a rubocop TODO.

@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: spec/jobs/content_delete_event_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  let(:third_user) { create(:user) }
+  let(:file_set) { create(:file_set, title: ['Hamlet'], user: user) }
+  let(:generic_work) { create(:generic_work, title: ['BethsMac'], user: user) }
+  let(:mock_time) { Time.zone.at(1) }
+  after do
+    Redis.current.keys('events:*').each { |key| Redis.current.del key }
+    Redis.current.keys('User:*').each { |key| Redis.current.del key }
+    Redis.current.keys('FileSet:*').each { |key| Redis.current.del key }
+    Redis.current.keys('GenericWork:*').each { |key| Redis.current.del key }
+  end
```
Comment (awead) [inline spec/jobs/content_delete_event_job_spec.rb]: What about putting this into a module under `spec/support` that's something like RedisCleaner.clean! and use it the same way we use ActiveFedora::Cleaner.clean!

File: spec/jobs/content_delete_event_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  it "logs the event to the depositor's profile and followers' dashboards" do
+    another_user.follow(user)
+    third_user.follow(user)
+    expect(user.profile_events.length).to eq(0)
+    expect(another_user.events.length).to eq(0)
+    expect(third_user.events.length).to eq(0)
+    expect(Time).to receive(:now).at_least(:once).and_return(mock_time)
+    event = { action: "User <a href=\"/users/#{user.to_param}\">#{user.user_key}</a> has deleted file '#{file_set.id}'", timestamp: '1' }
```
Comment (awead) [inline spec/jobs/content_delete_event_job_spec.rb]: Does it work to use :let here?

**Comment by jcoyne** [comment]
@awead I added a commit to refactor the existing tests using let blocks and change matchers.

**Comment by awead** [comment]
even better! :+1:

**Pull request merged**